src/resources/PackageResource.ts

```diff
@@ -28,12 +28,14 @@ export class PackageResource extends Resource {
   /**
    * Retrieves all packages (transactions).
    *
+   * @param params - Additional parameters for the query
```

Comment:
Are we going to document them here eventually or will consumers have to check out `Requests.GetAllPackagesParameters` themselves?

Reply:
The parameters will be documented in our documentation. At the same time, most IDEs should be able to direct users to the type definition in the library.